packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './controller';
+import template from './template.html';
+
+const component = {
+  bindings: {
+    onMoreInfoLinkClick: '&?',
```

Comment:
What is the use-case where this attribute would be required? I dont see it being used now... Probably, we can add it when needed?

Reply:
Yes it's used, triggered on the HTML part. 
`data-ng-click="$ctrl.onMoreInfoLinkClick()"`

Reply:
It's optional parameter (i added it for tracking purposes).

Reply:
removed

Reply:
removed.